src/main/java/com/juniorstart/juniorstart/payload/ChangeMailRequest.java

```diff
@@ -5,28 +5,30 @@
 import jakarta.validation.constraints.NotBlank;
 import lombok.Getter;
 
+import java.util.UUID;
+
 /** Class for method changeEmail from UserService
  * @author Dawid Wit
- * @version 1.0
+ * @version 1.1
  * @since 1.0
  */
 @Getter
 public class ChangeMailRequest {
     @Email
-    String email;
+    private final String email;
     @NotBlank
-    String name;
+    private final UUID privateId;
     @NotBlank
```

Comment:
Nie lepiej dać tutaj pola private same? 
Nie lepiej przekazać publicId?

Reply:
Dałem private Id, była dyskusja na ten tema z Adrianem chyba na general. Co do private final to zrobiłem zgodnie z uwagami powyżej. ```Zrób te pola private final I będzie cacy```

Reply:
Ustawienie private final to dobra praktyka w przypadku pól, które nie muszą być zmieniane. Zostawić private final.